python/fromdocs/kvoperations/readwrites/getatx.py

```diff
@@ -0,0 +1,40 @@
+from immudb import ImmudbClient
+
+URL = "localhost:3322"  # immudb running on your machine
+LOGIN = "immudb"        # Default username
+PASSWORD = "immudb"     # Default password
+DB = b"defaultdb"       # Default database name (must be in bytes)
+
+def main():
```

Comment:
this sample doesn't correspond to getAtTx but to getTxById

also, I'm curious about the get api, how it's differentiating from getAtRevision... in java sdk even with param overloading it was better to have a dedicated method get, getAtRevision, getAtTx ...